.github/scripts/userdata.sh

```diff
@@ -0,0 +1,136 @@
+#!/bin/bash
+
+set -e
+REPO_ROOT=`git rev-parse --show-toplevel`
+ES_VER=`$REPO_ROOT/bin/version-info --es`
+ODFE_VER=`$REPO_ROOT/bin/version-info --od`
+echo $ES_VER $ODFE_VER
+
+if [ "$#" -eq 0 ] || [ "$#" -gt 2 ] || [ -z "$1" ] || [ -z "$2" ]
+then
+    echo "Please assign 2 parameters when running this script"
+    echo "Format for dispatch event: \"client_payload\": {
+		\"distribution\" : \"rpm\",
+		\"security\" : \"enable\"
+	}"
+    echo "Example: $0 \"RPM\" \"ENABLE\""
+    echo "Example: $0 \"DEB\" \"DISABLE\""
+    exit 1
+fi
+
+###### RPM package with Security enabled ######
+if [ "$1" = "RPM" ]
+then
+cat <<- EOF > $REPO_ROOT/userdata_$1.sh
```

Comment:
Do we have to generate a script from another script like this?
Can you save a template then sed/awk replace the keywords?

Reply:
Too many replacements. The process and links are different for each distribution. Better to generate a temp file and pass that as userdata. So even if any customer wants to refer how to create a testing domain till will be easy to understand

Reply:
Resolved.